airbyte-integrations/connectors/source-bigquery/source_bigquery/source.py

```diff
@@ -26,165 +31,19 @@
 There are additional required TODOs in the files within the integration_tests folder and the spec.yaml file.
 """
 
-
-# Basic full refresh stream
-class BigqueryStream(HttpStream, ABC):
-    """
-    TODO remove this comment
-
-    This class represents a stream output by the connector.
-    This is an abstract base class meant to contain all the common functionality at the API level e.g: the API base URL, pagination strategy,
-    parsing responses etc..
-
-    Each stream should extend this class (or another abstract subclass of it) to specify behavior unique to that stream.
-
-    Typically for REST APIs each stream corresponds to a resource in the API. For example if the API
-    contains the endpoints
-        - GET v1/customers
-        - GET v1/employees
-
-    then you should have three classes:
-    `class BigqueryStream(HttpStream, ABC)` which is the current class
-    `class Customers(BigqueryStream)` contains behavior to pull data for customers using v1/customers
-    `class Employees(BigqueryStream)` contains behavior to pull data for employees using v1/employees
-
-    If some streams implement incremental sync, it is typical to create another class
-    `class IncrementalBigqueryStream((BigqueryStream), ABC)` then have concrete stream implementations extend it. An example
-    is provided below.
-
-    See the reference docs for the full list of configurable options.
-    """
-
-    # TODO: Fill in the url base. Required.
-    url_base = "https://example-api.com/v1/"
-
-    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
-        """
```

Comment:
why delete all of that ?

Reply:
This was the code automatically generated by the connector generator and following the pattern that I saw in other airbyte connectors it's better to put BigqueryStream class in `streams.py` file which you can find here https://github.com/artefactory/airbyte/blob/16c15fe1c810dabaafc57ec855c0d18a038c6654/airbyte-integrations/connectors/source-bigquery/source_bigquery/streams.py#L96 For the other two classes they were also generated automatically and we don't need them as they are just an example and we do not have those resources in Bigquery.